compliance/management/commands/decrypt_exports_inquiry.py

```diff
@@ -0,0 +1,76 @@
+"""
+Management command to decrypts a user's ExportInquiryLog record
+"""
+import sys
+
+from django.core.management import BaseCommand
+from django.contrib.auth import get_user_model
+from nacl.encoding import Base64Encoder
+from nacl.public import PrivateKey
+
+from compliance.api import decrypt_exports_inquiry
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Management command to decrypts a user's ExportInquiryLog record
+    """
+
+    help = "Decrypts a user's ExportInquiryLog record"
+
+    def add_arguments(self, parser):
+        """
+        Definition of arguments this command accepts
+        """
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument("--id")
```

Comment:
Can you add help text indicating that this is the `User` id and not the `ExportsInquiryLog` or other id?

Reply:
Also renamed it to `--user-id` so it's more obvious from the name